src/app/components/tabs.tsx

```diff
@@ -60,13 +59,13 @@ export function Tabs({
 }) {
   return (
     <Stack
-      position="relative"
-      isInline
-      borderRadius="12px"
       bg={color('bg-4')}
-      p="extra-tight"
-      width="100%"
+      borderRadius="10px"
       flexShrink={0}
+      isInline
+      p="extra-tight"
+      position="relative"
+      width="196px"
```

Comment:
Something here is odd. I am sorry I couldn't fully debug it to generate where the issue is, if you want we can share screen and take a look at it together:

<img width="375" alt="Screen Shot 2022-02-22 at 23 00 02" src="https://user-images.githubusercontent.com/1832037/155250664-af8bbb44-1843-4e2a-9b9f-18d49ca1f3b6.png">
.

Reply:
Can you specifically just draw on the image what is `odd` or provide more detail?

Reply:
yeah, I am sorry I should have been more specific:

The Balances and Activity list on the full-width view is narrow. On the extension view it takes the full width of the container.

This screen shot is taken on the extension view, where the bar should take the full width. This one shows it better:

<img width="936" alt="Screen Shot 2022-02-24 at 00 02 08" src="https://user-images.githubusercontent.com/1832037/155449883-e7793bbf-ee08-4837-ac0b-c5379339b3e3.png">

Reply:
@andresgalante actually, the design I was referencing for the home page changes showed it narrow in the popup, here: https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%90-Spec?node-id=2222%3A12791

And, also showed narrow here: https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%90-Spec?node-id=3%3A7

I do now have designs (new) from @landitus that show it full width here: https://www.figma.com/file/rphxUIsRqG3sLO1rxgJK6T/%F0%9F%8F%A0-Home?node-id=656%3A15585

So, I will change it but should these other places be updated? Not sure how that should work?

Reply:
Got it, the fact that there is no one source of truth for designs is a challenge that we'll overcome, but it'll take a bit of time to put things together.

For now, and until we have stronger processes around it let's rely on over communication, and maybe if we are working out of a design that has been done previews to Fernando's era, we can do a quick zoom sync to go over the designs before starting to code.

Reply:
Fixed this in the new redo PR so I'll resolve the conversation here. We can make sure to test it again there.